apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/service/AccessKeyService.java

```diff
@@ -0,0 +1,83 @@
+package com.ctrip.framework.apollo.biz.service;
+
+import com.ctrip.framework.apollo.biz.entity.AccessKey;
+import com.ctrip.framework.apollo.biz.entity.Audit;
+import com.ctrip.framework.apollo.biz.repository.AccessKeyRepository;
+import com.ctrip.framework.apollo.common.exception.BadRequestException;
+import java.util.List;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * @author nisiyong
+ */
+@Service
+public class AccessKeyService {
+
+  private static final int ACCESSKEY_COUNT_LIMIT = 5;
+
+  private final AccessKeyRepository accessKeyRepository;
+  private final AuditService auditService;
+
+  public AccessKeyService(
+      AccessKeyRepository accessKeyRepository,
+      AuditService auditService) {
+    this.accessKeyRepository = accessKeyRepository;
+    this.auditService = auditService;
+  }
+
+  public List<AccessKey> findByAppId(String appId) {
+    return accessKeyRepository.findByAppId(appId);
+  }
+
+  @Transactional(rollbackFor = Exception.class)
```

Comment:
Is it really necessary to specify `rollbackFor` here? As other parts of apollo codes don't have this and use default rollbackFor.

Reply:
It doesn't matter to specify`rollbackFor`, I add it cause I have installed `Alibaba Java Coding Guidelines` IDEA plugin, and the plugin advises me to add it.
![image](https://user-images.githubusercontent.com/8198862/70875910-8c066f00-1ff2-11ea-8791-aba3dee5acf0.png)
I can remove all `rollbackFor` to keep it as same as other codes.